src/annotator/components/test/ClusterToolbar-test.js

```diff
@@ -17,14 +18,36 @@ describe('ClusterToolbar', () => {
       />
     );
 
+  const toggleOpen = wrapper => {
+    act(() => {
+      wrapper
+        .find('button[data-testid="control-toggle-button"]')
+        .simulate('click');
```

Comment:
The `simulate` method uses `act()` internally and updates the wrapper when done. The `act` + `wrapper.update()` pattern is needed for when state updates are triggered outside of Enzyme wrapper methods (eg. a fetch resolves or a timeout expires or you call a function prop manually).

Reply:
OK, maybe, but this doesn't work without the `wrapper.update`

Reply:
The tests pass for me when I remove the `act(...)` callback and `wrapper.update()` call.

Reply:
🤷‍♂️ , maybe your Mac is haunted 👻